ui/cypress/support/commands.js

```diff
@@ -56,6 +56,18 @@ Cypress.Commands.add('setupMocks', () => {
     'api/prometheus/api/v1/query?query=node_uname_info',
     'fixture:prometheus/node-uname-info.json',
   ).as('getNodeUNameInfo');
+  cy.route(
+    'api/alertmanager/api/v2/alerts',
+    'fixture:alertmanager/alerts.json',
+  ).as('getAlerts');
```

Comment:
You'll likely need to rebase to make sure this doesn't conflict with what was already merged: https://github.com/scality/metalk8s/pull/2891/files#diff-c23023a5b2e52a3d6183689d6a32d9411ba3559297718fddd6068ee55e822875R143-R144